src/components/App/App.test.js

```diff
@@ -0,0 +1,5 @@
+const a = () => 1 + 2;
```

Comment:
Please, remove this useless one and create a valid one

Reply:
Removed!